4/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.8"
+services:
+  primarysilo:
+    build:  
+      context: .
+      dockerfile: ./ops/SiloHost/Dockerfile
+    environment:
+    - GATEWAYPORT=3000
+    - SILOPORT=2000
+    ports:
+      - "3000:8080"
+  secondarysilo:
+    build:  
+      context: .
+      dockerfile: ./ops/SiloHost/Dockerfile
+    environment:
+    - GATEWAYPORT=3000
```

Comment:
Port numbers should ideally not be shared between silos. Later on, we'll have ECS scenario where there could be more than one silo per EC2/Fargate host and they will share AWS IP and ports - in that scenario we'll have to port those shared values to something unique.

Reply:
Check this for reference: https://eateam.visualstudio.com/Emailage%20Code/_git/greetings-planet?path=%2Fsynopsis.md&version=GBmain&_a=preview&anchor=localhost-clustering